core/blockly.js

```diff
@@ -245,28 +256,32 @@ Blockly.defineBlocksWithJsonArray = function(jsonArray) {
               'Block definition #' + i + ' in JSON array' +
               ' overwrites prior definition of "' + typename + '".');
         }
-        Blockly.Blocks[typename] = {init: Blockly.jsonInitFactory_(elem)};
+        Blockly.Blocks[typename] = {init: jsonInitFactory(elem)};
       }
     }
   }
 };
+exports.defineBlocksWithJsonArray = defineBlocksWithJsonArray;
 
 /**
  * Is the given string a number (includes negative and decimals).
  * @param {string} str Input string.
  * @return {boolean} True if number, false otherwise.
  */
-Blockly.isNumber = function(str) {
+const isNumber = function(str) {
   return /^\s*-?\d+(\.\d+)?\s*$/.test(str);
 };
+exports.isNumber = isNumber;
 
 // Add a getter for Blockly.hueToHex, for legacy reasons.
-Object.defineProperty(Blockly, 'hueToHex', {
-  get: function() {
-    Blockly.utils.deprecation.warn(
-        'Blockly.hueToHex()', 'September 2021', 'September 2022',
-        'Blockly.utils.colour.hueToHex()');
-    return Blockly.utils.colour.hueToHex;
+Object.defineProperties(exports, {
+  hueToHex: {
+    get: function() {
+      deprecation.warn(
+          'Blockly.hueToHex()', 'September 2021', 'September 2022',
+          'Blockly.utils.colour.hueToHex()');
+      return colour.hueToHex;
+    }
   }
 });
```

Comment:
Only tangentially related to this PR, but since you're touching the accessors anyway:

Since there's no `set` accessor I'm sure that this accessor property does _not_ exist for the purpose of allowing library users to supply their own `hueToHex` implementation.  This raises the question: since `hueToHex` is just an ordinary function (rather than a formerly mutable export), why not provide a simple wrapper for it?:

```suggestion
/**
 * Convert a hue (HSV model) into an RGB hex triplet.
 * @deprecated Use Blockly.utils.co.our.hueToHex instead.
 * @param {number} hue Hue on a colour wheel (0-360).
 * @return {string} RGB code, e.g. '#5ba65b'.
 */
const hueToHex = function(hue) {
  Blockly.utils.deprecation.warn(
      'Blockly.hueToHex()', 'September 2021', 'September 2022',
      'Blockly.utils.colour.hueToHex()');
  return colour.hueToHex(hue);
};
exports.hueToHex = hueToHex;
```

Reply:
Updated accordingly.